### PR TITLE
Fixed grammatical errors in EXAMPLES.md

### PR DESCRIPTION
**Pull request by jonpsy**
Minor fixed in EXAMPLES.md

### CODE REVIEW

**Comment by jonpsy** [comment]
@gf712 , @vigsterkr  would you please review this?

File: doc/readme/EXAMPLES.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ For details, see `CMakeLists.txt` in `examples/meta/` for details, `generate.py`
 It is extremely simple to add a new example: simple create another `*.sg` file.
 We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).
 
-If you porting an example is great, even better is when it comes with integration testing data of the numerical output, as described in [DEVELOPING.md](DEVELOPING.md#testing).
+If you are porting a new example, great! Better yet would be to add the [data](https://github.com/shogun-toolbox/shogun-data) so that we can do integration tests automatically as described in [DEVELOPING.md](DEVELOPING.md#testing).
```
Comment (gf712) [inline doc/readme/EXAMPLES.md]: This should be “Better yet, add the data...” imo

**Review by gf712 — CHANGES_REQUESTED** [review]
Minor change

File: doc/readme/EXAMPLES.md (hunk 1 above)
Comment (jonpsy) [reply doc/readme/EXAMPLES.md]: That good too, any other corrections?

**Pull request merged**